types/uuidv4/index.d.ts

```diff
@@ -0,0 +1,34 @@
+// Type definitions for uuidv4 2.0
+// Project: https://github.com/thenativeweb/uuidv4#readme
+// Definitions by: Rindo Hinase <https://github.com/Hinaser>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare const uuidv4: UUIDv4;
+
+interface UUIDv4_is {
```

Comment:
It seems like this is a callable module with extra properties. If so, you should follow [this template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html).

Reply:
Hi @uniqueiniquity 
Thanks for reviewing!

I restructured my code following [the template](http://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html).

And I confirmed that new type definition works as intended.